proto/src/scim_v1/server.rs

```diff
@@ -216,6 +216,21 @@ pub enum ScimValueKanidm {
     KeyInternal(Vec<ScimKeyInternal>),
 }
 
+impl ScimEntryKanidm{
+    pub fn attr_str(&self, attr: &Attribute) -> Option<&str> {
+        match self.attrs.get(attr) {
+            Some(ScimValueKanidm::String(inner_string)) => Some(inner_string.as_str()),
+            Some(sv) => {
+                eprintln!("Scim entry did had the {attr} attr but expected ScimValueKanidm::String, actual: {sv:?}");
```

Comment:
No eprintln's or inline variables please 😄

```suggestion
                debug!("SCIM entry had the {} attribute but it was not a ScimValueKanidm::String type, actual: {:?}", attr, sv);
```